languages/swift/config.json

```diff
@@ -55,10 +55,10 @@
         "prerequisites": ["numbers", "conditionals"]
       },
       {
-        "slug": "strings",
+        "slug": "string-components",
         "uuid": "082d7aeb-ee25-4217-bd6f-82532a65797a",
-        "concepts": ["strings"],
-        "prerequisites": ["arrays"]
+        "concepts": ["string-componentss"],
```

Comment:
```suggestion
        "concepts": ["string-components"],
```

Reply:
Should there also be a concept here regarding the indexing into a string?

Reply:
Yes, I think it is important enough to make a separate concept. I also added an `opaque-indices` concept as those get introduced via string indices, which are opaque.